xwiki-platform-core/xwiki-platform-web/src/test/java/org/xwiki/web/DocumentTagsTest.java

```diff
@@ -75,7 +75,7 @@ void displayTagsWhenNoEditRightsAndTagPluginAvailableAndNoTags() throws Exceptio
         // - The tag label is displayed
```

Comment:
You need to edit this.

Reply:
you're still missing some comments adjustments here.

Reply:
can you tell me what else I should edit?

Reply:
@surli  Please check